### PR TITLE
Get synonyms and match text loosely

### PR DESCRIPTION
**Pull request by JesperDramsch**
Addresses #2

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+PyDictionary~=2.0.1
```
Comment (yagueto) [inline requirements.txt:1]: We currently have a different requirements file called (quite confusingly) [dev-requirements.txt](https://github.com/Yagueteiro/code-jam-2021/blob/main/dev-requirements.txt)

**Review by yagueto — CHANGES_REQUESTED** [review]
Could you merge the two files please?

File: requirements.txt (hunk 1 above)
Comment (JesperDramsch) [reply requirements.txt:1]: Usually you try to keep dev and user requirements separate. Are you sure?

Comment (yagueto) [reply requirements.txt:1]: Oh ok, I had never seen this file and assumed wrong lol

**Pull request merged**